### DISCUSSION TITLE
Calling All PostgreSQL Pros

### DISCUSSION THREAD

**Discussion by tinagrims87** [dbeaver/dbeaver#11186] [General]
Any recommendations for a paid or free SQL management tool? Currently, I'm working out of pgAdmin 4 4.11 but eventually (hopefully soon) I want to find a good, simple, reliable tool to use daily.

Recommendations and stories are welcome and anything helps!

**Comment by p91paul** [comment]
You are posting this question on the support forum of one such tool, so go with Dbeaver!

**Reply by flambchap** [reply]
Hello @p91paul. Our discussions are currently under attack by spambots. They post some DBMS-related questions from Reddit here, and after a while, include a spam link to their original post. It would be fantastic if you could report such posts to Github support when you encounter them. Thank you.

**Reply by p91paul** [reply]
How do I recognize it? this question just looks very strange, like an user who got lost... admittedly it's not the first one, but I didn't suspect a spam bot!

**Reply by flambchap** [reply]
Well, we didn't understand what's going on at first as well, but then @ShadelessFox just copied the entire contents of one of the spam posts and pasted it into Google, and found an exact word-by-word copy on Reddit as a first search result. We don't perform this type of search for every post here,  obviously, but this one triggered my attention because it just doesn't belong here. I can barely imagine a sane human person posting this type of question here.
